Andela-JUdensi/Inverted-Index
# Refactor the InvertedIndex application

#### What does this PR do?
1. Add a find on Github url application footer
2. Refactor the InvertedIndex class static methods
3. Rename the InvertedIndex class
4. Rename the InvertedIndex Test
5. Change Links on the index.html

#### Description of Task to be completed?
1. Link application to its Github repository
2. Add static methods to class
3. Rename files to meet convention
4. Change links on the html to reflect files changed

#### How should this be manually tested?
1. Visit the the [review app](idexii-staging.herokuapp.com) to see changes
2. View changes to the [review branch](https://github.com/Andela-JUdensi/Inverted-Index/tree/review)

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
